inc/auth.class.php

```diff
@@ -1033,7 +1024,7 @@ static function useAuthExt() {
     * @return boolean
    **/
    static function isAlternateAuth($authtype) {
-      return in_array($authtype, array(self::X509, self::CAS, self::EXTERNAL, self::API));
+      return in_array($authtype, array(self::X509, self::CAS, self::EXTERNAL));
```

Comment:
> Api is not an external auth, i think

This function is about "Alternate" not "External"... always the same confusion...